packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,490 @@
+    &:first-child .hds-form-super-select__option-group-title {
+      margin-top: 0;
+    }
+
+    // group separator line
+    &::before {
+      display: block;
+      height: 5px;
+      margin: 7px 5px 9px 5px; // top, right, bottom, left
+      border-bottom: 1px solid var(--token-color-border-primary);
+      content: "";
+    }
+
+    .hds-form-super-select__option-group {
+      padding-left: 16px;
+
+      &:first-child::before {
+        display: none;
+      }
+    }
+  }
+
+  .hds-form-super-select__option-group-title {
+    margin: 8px 0;
+    padding: 6px 7px 2px 15px; // top, right, bottom, left
+  }
+
+  // SEARCH
+  // (Single select, to be added to Multiple | in @beforeOptionsComponent area)
+  .ember-power-select-search {
+    margin-bottom: 3px;
+    padding: 11px 11px 0 11px;
+    border-radius: var(--token-form-control-border-radius);
+  }
+
+  .ember-power-select-search-input {
+    width: 100%;
+    min-height: $hds-super-select-item-height;
+    padding: var(--token-form-control-padding);
+    color: var(--token-form-control-base-foreground-value-color);
+    line-height: 1rem;
+    background-color: var(--token-form-control-base-surface-color-default);
+    border: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+    border-radius: var(--token-form-control-border-radius);
+    box-shadow: var(--token-elevation-inset-box-shadow);
+
+    // placeholder
+    &::placeholder {
+      color: var(--token-form-control-base-foreground-placeholder-color);
+    }
+
+    // status
+    &:hover,
+    &.mock-hover {
+      border-color: var(--token-form-control-base-border-color-hover);
+    }
+
+    // focus (same for all the states)
+    &:focus,
+    &.mock-focus {
+      border-color: var(--token-color-focus-action-internal);
+      // Notice: Safari doesn't apply a rounded border
+      outline: 3px solid var(--token-color-focus-action-external);
+      outline-offset: 0;
+    }
+
+    &[type="search"] {
+      padding-left: calc(var(--token-form-control-padding) + 24px); // extra space for the icon
+      background-image: var(--token-form-text-input-background-image-data-url-search);
+      background-repeat: no-repeat;
+      background-position: var(--token-form-text-input-background-image-position-x) 50%;
+      background-size: var(--token-form-text-input-background-image-size);
+
+      &::-webkit-search-cancel-button {
+        width: var(--token-form-text-input-background-image-size);
+        height: var(--token-form-text-input-background-image-size);
+        background-image: var(--token-form-text-input-background-image-data-url-search-cancel);
+        background-position: center center;
+        background-size: var(--token-form-text-input-background-image-size);
+        appearance: none;
+      }
+    }
+  }
+
+  // MULTIPLE
+  // Tag List (Note: May be replaced with HDS Tag component in the future)
+  .ember-power-select-multiple-options {
```

Comment:
@I _think_ you need to add a `min-width: 0` here or this happens when a parent has a width and the option "tag" has a very long text:

<img width="340" alt="screenshot_3711" src="https://github.com/hashicorp/design-system/assets/686239/a8616d8b-50c3-47c5-878d-8bd9bcbe9af9">

```suggestion
  .ember-power-select-multiple-options {
      min-width: 0;
```